### PR TITLE
Update `$app` modifier

### PR DESCRIPTION
**Pull request by grumaxxx**

### CODE REVIEW

File: docs/general/ad-filtering/create-own-filters.md [hunk 1]
```diff
@@ -233,6 +233,10 @@ This modifier lets you narrow the rule coverage down to a specific application (
 
 > For Mac, you can find out the bundle ID or the process name of the app by viewing the respective request details in the Filtering log.
 
+> **Compatibility with different versions of AdGuard**
+>
+> Starting with CoreLibs v1.12, the `$app` modifier for Windows is case-insensitive.
```
Comment (ameshkov) [inline docs/general/ad-filtering/create-own-filters.md:238]: This does not belong to the compatibility section. Just note that the names are case-insensitive (you can add a note that this behavior is from v1.12).

**Review by ameshkov — CHANGES_REQUESTED** [review]

File: docs/general/ad-filtering/create-own-filters.md [hunk 2]
```diff
@@ -233,9 +233,7 @@ This modifier lets you narrow the rule coverage down to a specific application (
 
 > For Mac, you can find out the bundle ID or the process name of the app by viewing the respective request details in the Filtering log.
 
-> **Compatibility with different versions of AdGuard**
->
-> Starting with CoreLibs v1.12, the `$app` modifier for Windows is case-insensitive.
+> For Windows, starting with CoreLibs v1.12, the `$app` modifier for Windows is case-insensitive.
```
Comment (ameshkov) [inline docs/general/ad-filtering/create-own-filters.md:236]: Again, this is a part of the modifier definition, why a separate blockquote?

```
* Windows — use the process name, e.g. `chrome.exe`. Note, that the process name is case-insensitive starting with CoreLibs v1.12.
```

**Review by ameshkov — CHANGES_REQUESTED** [review]

File: docs/general/ad-filtering/create-own-filters.md (hunk 1 above)
Comment (slavaleleka) [reply docs/general/ad-filtering/create-own-filters.md]: actually we do write such notes like this

File: docs/general/ad-filtering/create-own-filters.md (hunk 2 above)
Comment (grumaxxx) [reply docs/general/ad-filtering/create-own-filters.md:236]: Just for attention, similar to the Mac note. OK, I'll redo it.

**Review by slavaleleka — APPROVED** [review]

**Review by Alex-302 — APPROVED** [review]

File: docs/general/ad-filtering/create-own-filters.md [hunk 3]
```diff
@@ -228,7 +228,7 @@ The following modifiers are the most simple and frequently used. Basically, they
 This modifier lets you narrow the rule coverage down to a specific application (or a list of applications). This might be not too important on Windows and Mac, but this is very important on mobile devices where some of the filtering rules must be app-specific.
 
 * Android — use the app package name, e.g. `org.example.app`.
-* Windows — use the process name, e.g. `chrome.exe`.
+* Windows — use the process name, e.g. `chrome.exe`. Note, that the process name is case-insensitive starting with CoreLibs v1.12.
```
Comment (el-termikael) [inline docs/general/ad-filtering/create-own-filters.md:231]:
```suggestion
* Windows — use the process name, e.g. `chrome.exe`. Note that the process name is case-insensitive starting with CoreLibs v1.12.
```

**Review by el-termikael — CHANGES_REQUESTED** [review]

**Review by el-termikael — APPROVED** [review]

**Pull request merged**